SpatialGDK/Documentation/content/pricing-and-support/versioning-scheme.md

```diff
@@ -0,0 +1,93 @@
+
+An alternative option is to use the `preview` branch. This branch contains the latest `*-preview` version, which we test using Improbable partner projects, but not as extensively as versions in `release`. Preview releases are for teams already developing on the GDK and working closely with Improbable. 
+
+In summary:
+
+<table>
+  <tr>
+   <td><strong>Branch</strong>
+   </td>
+   <td><strong>Update frequency</strong>
+   </td>
+   <td><strong>Maturity</strong>
+   </td>
+   <td><strong>Support policy</strong>
+   </td>
+  </tr>
+  <tr>
+   <td><code>release</code>
+   </td>
+   <td>A few times per year (as part of SpatialOS for Unreal milestones such as 2019.1).
+   </td>
+   <td>Fully tested, stable, and documented.
+   </td>
+   <td>Fully supported via our forums and Discord.
+   </td>
+  </tr>
+  <tr>
+   <td><code>preview</code>
+   </td>
+   <td>~Monthly
+   </td>
+   <td>Minimal tested focused on partner projects. Documentation is incomplete.
+   </td>
+   <td>Support is only provided to customers on the paid support tier.
+   </td>
+  </tr>
+</table>
+
+### Unreal Engine Fork branches
+
+The [Unreal Engine Fork](https://github.com/improbableio/UnrealEngine) follows the versioning and branches pattern described above, but the branch names have `X.XX-SpatialOSUnrealGDK` following prepended to them (where `X.XX` is the Unreal Engine version of the fork.)
+
+For example:
+
+*   ``4.22-SpatialOSUnrealGDK-release`` (default)
+*   ``4.22-SpatialOSUnrealGDK-preview``
+
+when Unreal Engine 4.22 is the version supported.
+
+## Unreal Engine version support
+
+**The GDK has full support for one version of Unreal Engine at a time** - the version that is shown in the Unreal Engine Fork repository’s default branch name, and is indicated in the Get Started documentation.
+This is the only version that the Example Project supports. With version 0.6.0, this is Unreal Engine version 4.22. 
+
+To enable you to transition, the GDK has legacy support during one release cycle for the Unreal Engine version previously supported. For example, with version 0.6.0, the legacy support is for Unreal Engine 4.20, which is available through the ``4.20-SpatialGDK-release`` branch of the SpatialOS Engine Fork. The Example Project is no longer supported with this version.
+
+We will usually start working on supporting the most recent Unreal Engine minor version after a few patch releases that make that minor version more stable. For instance, we will start working on Unreal Engine 4.23 support for our next GDK release once 4.23.2 or 4.23.3 is available. To see when a new version will be supported, check out the [Development Roadmap](https://github.com/spatialos/unrealgdk/projects/1).
```

Comment:
this is non-comittal and not helpful, we should either be declarative (e.g: We endeavour to support the latest stable and recommended version of the Unreal Engine less than 3 months after it's released) or not include this at all.

Reply:
I agree that's better language, updating to that. 
It still is non-commital - I disagree that a non-commital statement is not helpful